02.calendar/calendar.js

```diff
@@ -0,0 +1,43 @@
+function printCalendar(month, year){
+    printHeader(month, year);
+    const first_day_of_month = new Date(year, month -1, 1);
+    const first_wday = first_day_of_month.getDay();
+    const last_day_of_month = new Date(year, month, 0);
+    const dates = getDates(first_day_of_month, last_day_of_month);
+    let str =  '   '.repeat(first_wday) + (first_wday > 2 ? ' '.repeat(first_wday - 2) : '')
+    dates.forEach(day => {
+        if(day.getDay() !== 0 && day !== first_day_of_month){
```

Comment:
if文は

```suggestion
        if (day.getDay() !== 0 && day !== first_day_of_month) {
```

のようにスペースを入れることが多いです。

前回のレビューでお伝えした https://beautifier.io/ は使ってみましたか？
こういったツールを使ってコード全体のフォーマットを見直してみてください〜。

Reply:
すみませんでした、前回はまだ使ってないです。修正しました。